airflow/api_connexion/openapi/v1.yaml

```diff
@@ -88,13 +88,13 @@ paths:
       - $ref: '#/components/parameters/ConnectionID'
 
     get:
-      summary: Get a connection entry
+      summary: Get a connection
       x-openapi-router-controller: airflow.api_connexion.endpoints.connection_endpoint
       operationId: get_connection
       tags: [Connection]
       responses:
         '200':
-          description: Successful response.
+          description: Sucess.
```

Comment:
In L49 we use more meaningful description. Should we unify the style?

Reply:
Right. This field should describe **when** this response is returned and not contain information on what the field contains. We have other places to describe the content.

Reply:
I update the description.